nuxeo-admin-console-web/angular-app/src/app/features/sub-features/generic-multi-feature-layout/components/folder-tab/folder-tab.component.ts

```diff
@@ -184,13 +184,33 @@ export class FolderTabComponent implements OnInit, OnDestroy {
     document.getElementById("inputIdentifier")?.focus();
   }
 
+  isIdAndPathRequired(activeFeature: string): boolean {
+    return !featuresRequiringOnlyId.includes(activeFeature);
+  }
+
   getErrorMessage(): string | null {
-    if (this.inputForm?.get("inputIdentifier")?.hasError("required")) {
+    const areIdAndPathRequired = this.isIdAndPathRequired(this.activeFeature as string);
+    if (this.inputForm?.get("inputIdentifier")?.hasError("required") && !areIdAndPathRequired) {
```

Comment:
make the condition - this.inputForm?.get("inputIdentifier")?.hasError("required")  common

Reply:
done